### PR TITLE
iam beta docs

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: beta/iam/guides/how-to-messaging-delegate.md
Comment (calebk1) [inline beta/iam/guides/how-to-messaging-delegate.md:74]: Also no Java.

File: beta/iam/public-classes-enums.md
Comment (calebk1) [inline beta/iam/public-classes-enums.md:11]: Since this is the first mention of Adobe Journey Optimizer within this document, it should be referred to as its full name, as such "... Adobe Journey Optimizer (AJO) Campaign."

File: beta/iam/setup-sdk.md
Comment (calebk1) [inline beta/iam/setup-sdk.md:1]: "Set up AEPMessaging SDK"

File: beta/iam/setup-sdk.md
Comment (calebk1) [inline beta/iam/setup-sdk.md:3]: No need to put Beta in all caps.

**Review by calebk1 — CHANGES_REQUESTED** [review]
A few changes to make.

Additionally:

1. Remove the .DS_Store changes, if possible (unless you're just removing it, which then would be fine).
2. All instances of JavaScript should be written like JavaScript.
3. Instead of making the tabs Swift and Java, it should be iOS and Android, with the languages used underneath as headers.
4. It seems that there's no Android information at all? If it isn't available, I'd just remove it entirely, instead of giving users the false hope/promise that there are Java things available and there just being an empty tab.

Thanks!

File: beta/iam/edge-and-launch-configuration.md
Comment (sbenedicadb) [reply beta/iam/edge-and-launch-configuration.md:5]: did you mean "please [follow the] tutorial on creating a datastream"?

**Comment by calebk1** [comment]
With regards to the following comments:

1. Sweet! Just wanted to make sure, but removing it should be a-okay.
2. Oh! There some instances where it was written as javascript or Javascript, but, if I recall correctly, it's preferred to style it as JavaScript.
3. I'd just leave it as a sentence instead. I don't think it really needs any heading, since it seems like it's just a one step process, nor is it really a header with any subsequent content.
4. Got it! No worries then! I was just worried some wires got crossed in the process.

File: beta/iam/guides/how-to-call-javascript-from-native.md
Comment (sbenedicadb) [reply beta/iam/guides/how-to-call-javascript-from-native.md:37]: coming soon :)

**Comment by sbenedicadb** [comment]
updated!  let me know if there's anything else.

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: beta/iam/public-classes-enums.md
Comment (sbenedicadb) [reply beta/iam/public-classes-enums.md:198]: the goal here was just to provide a full definition of the enum.  the `toString` method, in conjunction with the table of string values, will allow the reader to determine the result of calling `toString` for each enum value.

File: SUMMARY.md
Comment (sbenedicadb) [reply SUMMARY.md:191]: no!  good catch, i'll add it.

File: beta/iam/guides/how-to-messaging-delegate.md
Comment (sbenedicadb) [reply beta/iam/guides/how-to-messaging-delegate.md:94]: this is intentional.  markdown will always order the numbered list correctly.  we use all `1` values so that if a new item needs to be inserted in the middle of the list, you don't ever have to worry about updating the numbers.

File: beta/iam/edge-and-launch-configuration.md
Comment (calebk1) [reply beta/iam/edge-and-launch-configuration.md]: I somehow didn't comment on this, but yes, I did mean please follow the. However, I'd wrap the URL block around tutorial on creating a datastream still.

File: beta/iam/guides/how-to-messaging-delegate.md
Comment (sbenedicadb) [reply beta/iam/guides/how-to-messaging-delegate.md:190]: in swift, yeah - but the `webView` property in `FullscreenMessage` isn't exposed to objc

**Comment by sbenedicadb** [comment]
thank you @swarna04  - updates have been made

**Comment by sbenedicadb** [comment]
@calebk1 @swarna04 we can merge this in once you all give final approval

File: beta/iam/public-classes-enums.md
Comment (swarna04) [reply beta/iam/public-classes-enums.md:198]: Alright, let's keep it as is for now. Hopefully, we'd be able to move such items to developer docs on extension GitHub repo soon.

File: beta/iam/guides/how-to-messaging-delegate.md
Comment (swarna04) [reply beta/iam/guides/how-to-messaging-delegate.md:94]: Sounds good, didn't know about this 👀

**Review by swarna04 — APPROVED** [review]

**Pull request merged**
